src/gobworkflow/config.py

```diff
@@ -2,18 +2,23 @@
 
 MESSAGE_BROKER = os.environ["MESSAGE_BROKER_ADDRESS"]
 
-WORKFLOW_QUEUE = "gob.workflow.proposal"
-LOG_QUEUE = "gob.log.all"
+WORKFLOW_QUEUE = "gob.workflow"
+LOG_QUEUE = "gob.log"
 
 QUEUES = [
     {
         "exchange": "gob.workflow",
```

Comment:
Why not use WORKFLOW_QUEUE here as the name of the exchange?
Or rename WORKFLOW_QUEUE to WORKFLOW_EXCHANGE and use that value?
It prevents the duplication of the "gob.workflow" and "gob.log" strings

Reply:
Was de variabele naam die er al stond, ik pas het aan.